winml/test/common/googleTestMacros.h

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
+#define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING
```

Comment:
Which part of test code needs to silence deprecation warnings? Does this also need to be in taefTestMacros.h?

Reply:
Line 87. Not really needed for now since warnings as errors aren't enabled, but I'll make another PR in the future fixing remaining warnings.